assembl/graphql/schema.py

```diff
@@ -511,6 +515,13 @@ class QuestionInput(graphene.InputObjectType):
     title_entries = graphene.List(LangStringEntryInput, required=True)
 
 
+# How the file upload works
+# With the https://github.com/jaydenseric/apollo-upload-client
```

Comment:
There is also no "cleanup" cron job that removes attachments on posts that no longer exist. This was by design. Certainly now we should have a discussion about it....